WordPress/Classes/ViewRelated/Blog/My Site/NoSitesView.swift

```diff
@@ -28,12 +28,12 @@ struct NoSitesView: View {
                 .edgesIgnoringSafeArea(.all)
 
             mainView
-                .padding(.horizontal, Length.Padding.medium)
+                .padding(.horizontal, Length.Padding.large)
```

Comment:
We have updated the padding names:

`small` -> `medium`
`medium -> `large`
`large` -> `max`